### DISCUSSION TITLE
DB view freezing [Apr 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5462] [Discord Threads]
> 🙋‍♂️ **Jakob from bunch**  
#### anything that you can take away from these errors that leads this issue?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1081222236939235489/Screenshot_2023-03-03_at_15.21.55.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **Jakob from bunch**  
#### <@446452643221602304> could you maybe help with this? 🙏  
#  
> 🙋‍♂️ **kyleecodes**  
#### Not sure, the DOMException is a common JavaScript issue and it could be messing with your request, tagging <@&682859083799199814>. Can you post your Project Info?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1081222236888907879)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
